tests/unit_tests/db_engine_specs/test_doris.py

```diff
@@ -85,25 +86,25 @@ def test_get_column_spec(
         (
             "doris://user:password@host/db1",
             {"param1": "some_value"},
-            "db1",
+            "internal.information_schema",
```

Comment:
Is this a breaking change? To me it seems `db1` is indeed the database, not `internal.information_schema`?

Reply:
@villebro  Yes, in Apache Doris, in order to be compatible with different BI tools, we have information_schema for each catalog. When no catalog is specified, the default catalog is internal. This feature corresponds to this PR of Doris, https://github.com/apache/doris/pull/28919

Reply:
Got it, thanks for the clarification 👍